app/controllers/my-tickets/upcoming.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+export default Controller.extend({
+  actions: {
+    shareEvent(event) {
```

Comment:
where is this action being used @shreyanshdwivedi

Reply:
@CosmicCoder96 this action was used in previous version of `My Tickets`, however it was not being used. 
I think it can be used later which will provide functionality of sharing tickets from this section.